strings/charoccurrence/charoccurrence.go

```diff
@@ -0,0 +1,19 @@
+// charoccurrence.go
+// description: Chars occurrence in the sentence
+// details: An algorithm to count chars in string
+// author(s) [Moein](https://github.com/mo1ein)
+// see charoccurrence_test.go
+
+package charoccurrence
+
+func Count(text string) map[rune]int {
```

Comment:
Personally I feel this is too trivial; what do you guys think? @siriak @raklaptudirm @yanglbme

Reply:
Please don't mark things as resolved when they haven't been resolved 🙂

Reply:
@tjgurwara99, it's too simple as an algorithm, but can be a good entry point for somebody who only starts exploring this repo.

Reply:
I think we may include it taking into account that tests, description and everything else is in place